templates/engage/thank-you.html

```diff
@@ -35,39 +35,49 @@ <h1>
 </section>
 
 <section class="p-strip">
-  <div class="row">
-    <div class="col-8">
+  <div class="u-fixed-width">
+  {% if 'pages.ubuntu.com' or 'assets.ubuntu.com' in resource_url %}
+    {% if form_details and "access_to_content" in metadata and metadata.access_to_content == "true" %}
+    
+      <h3>We've emailed a copy of {{ resource_name }} to {{ form_details.email }}</h3>
+      <p>
+        <a class="p-button--positive" href="{{ request_url }}">Back to last page</a>
+        <a class="p-button" href="/contact-us">Contact us</a>
+      </p>          
       <p>
-        {% if "thank_you_text" in metadata %}
-          {{ metadata["thank_you_text"] }}
-        {% else %}
-          The {{ resource_name }} is now ready to download.
-        {% endif %}
+        Not received it? Check your spam folder and that you've used the right email address. <a href="{{ request_url }}">Or try resending it</a>
```

Comment:
According to Miro, "Or try sending" should go to the next line? 

<img width="833" alt="image" src="https://user-images.githubusercontent.com/57550290/191966875-ffe0e3bd-ce6d-47b3-be9e-3597105365ab.png">

Reply:
Maybe, but we do have a Vanilla paragraph limit